notebooks/itslive/itslive.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div align=\"center\"> \n",
+    "<img src='./img/header.png'/>\n",
+    "</div>\n",
+    "\n",
+    "## [Global Ice Velocities](https://its-live.jpl.nasa.gov/)\n",
+    "    \n",
+    "The Inter-mission Time Series of Land Ice Velocity and Elevation (ITS_LIVE) project facilitates ice sheet, ice shelf and glacier research by providing a globally comprehensive and temporally dense multi-sensor record of land ice velocity and elevation with low latency.\n",
+    "\n",
+    "Scene-pair velocities generated from satellite optical and radar imagery.\n",
+    "\n",
+    "* Coverage: All land ice\n",
+    "* Date range: 1985-present\n",
+    "* Resolution: 240m\n",
+    "* Scene-pair separation: 6 to 546 days\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "* If you want to query our API directly using  your own software here is the OpenApi endpoint https://staging.nsidc.org/apps/itslive-search/docs\n",
```

Comment:
This is ready to merge with the API still in staging? Does this need to point to a PROD endpoint instead?